### PR TITLE
Add 'Abstract section' in a post

### PR DESCRIPTION
**Pull request by alvarogarcia7**
Please wait for merging, until @TobyRet has approved the suggested abstract

### CODE REVIEW

**Comment by alvarogarcia7** [comment]
@TobyRet has now approved this comment

File: _includes/related_posts.html [hunk 1]
```diff
@@ -51,6 +58,13 @@
 					{% assign author = relevantPostAttributes[5] %}
 					{% assign postContent = relevantPostAttributes[6] %}
 					{% assign postDate = relevantPostAttributes[7] %}
+					{% assign abstract = relevantPostAttributes[8] %}
+					
+					{% if abstract %}
```
Comment (Gryff) [inline _includes/related_posts.html:63]: @alvarogarcia7 Just to double check, relevantPostAttributes[8] doesn't error if there is nothing there? So all other posts without abstracts aren't affected.

Comment (alvarogarcia7) [inline _includes/related_posts.html:63]: @Gryff good eye. it does not fail, but `abstract` is null/empty and does not pass the check on line 63: `if abstract`.

Comment (Gryff) [inline _includes/related_posts.html:63]: Awesome, thanks :)

Comment (Gryff) [inline _includes/related_posts.html:63]: @alvarogarcia7 One last thing is to check with @halimakoundi as there may be some merge conflicts here?

Comment (ghost) [inline _includes/related_posts.html:63]: @alvarogarcia7 and @Gryff, the change I made in that file is at line 5, so git should be able to merge those two without issues.. right?

Comment (Gryff) [inline _includes/related_posts.html:63]: Sounds correct to me :)

**Pull request merged**